### PR TITLE
feat(@wallet): move crypto on ramp async

### LINKED ISSUE #9491: Remove call to wallet after wallet is enable
Ensure all call to wallet are done only when wallet is enabled:
[bootstrap perf](https://github.com/status-im/status-desktop/issues/9193#issuecomment-1416031016)

### PR DESCRIPTION
**Pull request by alaibe**
@jrainville the only query left is getEthereumChains which is used by browser and not maintained

fixes #9491

### CODE REVIEW

File: src/app_service/service/wallet_account/service.nim [hunk 1]
```diff
@@ -416,7 +416,7 @@ QtObject:
 
   method toggleTestNetworksEnabled*(self: Service) =
     discard self.settingsService.toggleTestNetworksEnabled()
-    self.tokenService.init()
+    self.tokenService.loadData()
```
Comment (jrainville) [inline src/app_service/service/wallet_account/service.nim:419]: You already have a `signalConnect` that does the same thing, so you can remove one of them

**Review by jrainville — COMMENTED** [review]
Looks good. Do you think there would be a way to make it lazy load as well? ie only call them when we enter the wallet section?

File: src/app_service/service/wallet_account/service.nim (hunk 1 above)
Comment (alaibe) [reply src/app_service/service/wallet_account/service.nim:419]: nope the signal connect is when the wallet is enaled, this toggle is when test networks are enabled

**Comment by alaibe** [comment]
@jrainville we are planning to get rid of the token service fully, cause the token we have here contains no information about the account, so we don't really need it. So if you are ok, we will just remove this service in one or 2 cycle?

**Review by jrainville — APPROVED** [review]

**Review by Khushboo-dev-cpp — APPROVED** [review]

**Pull request merged**